lib/emque/producing/publisher/google_cloud_pubsub.rb

```diff
@@ -0,0 +1,47 @@
+require "google/cloud/pubsub"
+
+module Emque
+  module Producing
+    module Publisher
+      class GoogleCloudPubsub < Emque::Producing::Publisher::Base
+        def initialize(project_id:, credentials:)
+          self.pubsub = Google::Cloud::PubSub.new(
+            :project_id => project_id
+            # :credentials => credentials # only works when reading directly from ENV
```

Comment:
This is still being super flakey when passing credentials directly. If the google library accesses ENV directly it works, but if you specifically pass in the ENV value, it doesn't work.